packages/main/src/plugin/container-registry.ts

```diff
@@ -2432,24 +2434,59 @@ export class ContainerProviderRegistry {
     }
   }
 
+  protected async isTarPlayBuildSupported(internalProvider: InternalContainerProvider): Promise<boolean> {
+    if (!internalProvider.api || !internalProvider.libpodApi) {
+      throw new Error('No provider with a running engine');
+    }
+
+    const version = await internalProvider.api.version();
+
+    // let's nicely format it
+    const coerced = coerce(version.Version);
```

Comment:
This section I'm a bit confused on haha. Maybe a comment be added. Do we "fake" that it's version 5.3.0 or something? I feel like 5.3.0 should be a constant somewhere instead?

Reply:
ACK!